serverless/installing_serverless/installing-openshift-serverless.adoc

```diff
@@ -22,24 +22,35 @@ The total size requirements to run {ServerlessProductName} are dependent on the
 
 In the size requirement provided, an application can scale up to 10 replicas. Lowering the actual CPU request of applications can increase the number of possible replicas.
 
-For more advanced use-cases such as logging, monitoring, or metering on {product-title}, you must deploy more resources. Recommended requirements for such use-cases are 24 CPUs and 96GB of memory.
-
 You can use the MachineSet API to manually scale your cluster up to the desired size. The minimum requirements usually mean that you must scale up one of the default MachineSets by two additional machines.
 
 For more information on using the MachineSet API, see the documentation on xref:../../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-aws[Creating MachineSets].
 
 For more information on scaling a MachineSet manually, see the documentation on xref:../../machine_management/manually-scaling-machineset.adoc#manually-scaling-machineset[manually scaling MachineSets].
 
-// Check requirements for HA and whether we need to up the requirements for memory etc.
-
 [NOTE]
 ====
 The requirements provided relate only to the pool of worker machines of the {product-title} cluster. Master nodes are not used for general scheduling and are omitted from the requirements.
 ====
 
+[NOTE]
+====
+The following limitations apply to all {ServerlessProductName} deployments:
+
+* Maximum number of Knative services: 1000
+* Maximum number of Knative revisions: 1000
+====
+
+=== Additional requirements for advanced use-cases
+
+For more advanced use-cases such as logging or metering on {product-title}, you must deploy more resources. Recommended requirements for such use-cases are 24 CPUs and 96GB of memory.
+
+// If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200m - 2GB of memory for each replica of the Knative Serving control plane.
```

Comment:
@abrennan89 just checking if you needed these for anything before we merge?

Reply:
Nope, not until tomorrow :) I have a PR to make everything like this live for GA when it's ready to go